docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml

```diff
@@ -35,9 +35,9 @@ content:
     weight: 6
     label: above
     settings:
-      mirador_settings: ''
+      link_to_entity: false
     third_party_settings: {  }
-    type: mirador_text
```

Comment:
This doesn't look right. We changing the field formatter type from Mirador to simple text? Is this intended?